Algorithms/Dynamic-Programming/LCS.cpp

```diff
@@ -3,112 +3,87 @@
 
 // Author: Pritam Negi.
 
-#include<bits/stdc++.h>
+#include <bits/stdc++.h>
 using namespace std;
 
-// Declaration of global variables
-// Program supports two string of upto 10000 characters.
-// 2D Array c is used to store the length of LCS
-// 2D Array b stores the direction of tracing the array c at a particular point to find the LCS.
-int i, j, m, n, c[10010][10010];
-char b[10010][10010];
-string x, y;
+string s1, s2;
```

Comment:
Include a short description of what this program does in brief at the very beginning of the file.